server/checks_test.go

```diff
@@ -130,49 +130,160 @@ func TestListSecurityChecks(t *testing.T) {
+
+			_, err = managementClient.Default.SecurityChecks.ChangeSecurityChecks(params)
+			require.NoError(t, err)
+
+			resp, err = managementClient.Default.SecurityChecks.ListSecurityChecks(nil)
+			require.NoError(t, err)
+			require.NotEmpty(t, resp.Payload.Checks)
+
+			for _, c := range resp.Payload.Checks {
+				if c.Name == check.Name {
+					assert.Equal(t, !check.Disabled, c.Disabled)
+					break
+				}
+			}
+		}
 	})
-	require.NoError(t, err)
-	assert.True(t, res.Payload.Settings.SttEnabled)
 
-	resp, err := managementClient.Default.SecurityChecks.ListSecurityChecks(nil)
-	require.NoError(t, err)
-	require.NotEmpty(t, resp.Payload.Checks)
+	t.Run("change interval error", func(t *testing.T) {
+		defer restoreSettingsDefaults(t)
+		// Enable STT
+		res, err := client.ChangeSettings(&server.ChangeSettingsParams{
+			Body: server.ChangeSettingsBody{
+				EnableStt: true,
+			},
+			Context: pmmapitests.Context,
+		})
+		require.NoError(t, err)
+		assert.True(t, res.Payload.Settings.SttEnabled)
 
-	var check *security_checks.ChecksItems0
-	var params *security_checks.ChangeSecurityChecksParams
+		resp, err := managementClient.Default.SecurityChecks.ListSecurityChecks(nil)
+		require.NoError(t, err)
+		require.NotEmpty(t, resp.Payload.Checks)
+		assert.Equal(t, "STANDARD", *resp.Payload.Checks[0].Interval)
+
+		var check *security_checks.ChecksItems0
+		var params *security_checks.ChangeSecurityChecksParams
 
-	// enable ⥁ disable loop, it checks current state of first returned check and changes its state,
-	// then in second iteration it returns state to its origin.
-	for i := 0; i < 2; i++ {
 		check = resp.Payload.Checks[0]
+		interval := "unknown_interval"
 		params = &security_checks.ChangeSecurityChecksParams{
 			Body: security_checks.ChangeSecurityChecksBody{
 				Params: []*security_checks.ParamsItems0{
 					{
-						Name:    check.Name,
-						Disable: !check.Disabled,
-						Enable:  check.Disabled,
+						Name:     check.Name,
+						Interval: &interval,
 					},
 				},
 			},
 			Context: pmmapitests.Context,
 		}
 
 		_, err = managementClient.Default.SecurityChecks.ChangeSecurityChecks(params)
+		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "unknown value \"\\\"unknown_interval\\\"\" for enum management.SecurityCheckInterval")
+	})
+
+	t.Run("change interval normal", func(t *testing.T) {
+		defer restoreSettingsDefaults(t)
+		defer restoreCheckIntervalDefaults(t)
+		// Enable STT
+		res, err := client.ChangeSettings(&server.ChangeSettingsParams{
+			Body: server.ChangeSettingsBody{
+				EnableStt: true,
+			},
+			Context: pmmapitests.Context,
+		})
 		require.NoError(t, err)
+		assert.True(t, res.Payload.Settings.SttEnabled)
+
+		resp, err := managementClient.Default.SecurityChecks.ListSecurityChecks(nil)
+		require.NoError(t, err)
+		require.NotEmpty(t, resp.Payload.Checks)
+		assert.Equal(t, "STANDARD", string(*resp.Payload.Checks[0].Interval))
+
+		var params *security_checks.ChangeSecurityChecksParams
```

Comment:
let's remove these two variables from here and replace `params =` with `params :=` below

Reply:
here and in other places where it's possible